src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -861,39 +861,41 @@ description: (
 
 reference:
       (
-            PACKAGE (DOT | CLASS)* HASH? MEMBER? parameters?
-            | (DOT | CLASS)+ HASH? MEMBER? parameters?
+            PACKAGE_CLASS HASH? MEMBER? parameters?
```

Comment:
Actually, the javadoc tool is able to recognize package.Class.NestedClassAtDepth1.NestedClassAtDepth2#member kind of constructs. So we can't recognize class.